tutorials/Tutorial5_Evaluation.py

```diff
@@ -204,6 +203,57 @@ def tutorial5_evaluation():
     )
     pipeline.print_eval_report(eval_result_with_upper_bounds)
 
+    # ## Advanced Label Scopes
+    # Usually answers are considered as correct if the predicted answer matches the gold answer in the labels. Documents are considered correct if the predicted document ID matches the gold document ID in the labels. Sometimes these simple definitions of "correctness" are not sufficient. There are cases where you want to further specify the "scope" within which an answer or a document is considered as correct. For this reason `EvaluationResult.calculate_metrics()` offers the parameters `answer_scope` and `document_scope`.
+    #
+    # For example imagine you want to ensure that an answer is only considered as correct if it stems from a specific context of surrounding words. This is especially useful if your answer is very short, like a date (e.g. "2011") or a place (e.g. "Berlin"). This short answer might easily appear in multiple completely different contexts. Some of those contexts might perfectly fit the actual question and answer it. Some others might not: they don't relate to the question at all but still contain the answer string. In that case you might want to ensure that only those answers that stem from the correct context should actually be considered as a correct answer. To do that, you specify `answer_scope="context"` in `calculate_metrics()`.
+    #
+    # There are the following values for `answer_scope`:
+    # - `any` (default): Any matching answer is considered as correct.
+    # - `context`: The answer is only considered as correct if its context matches as well. Uses fuzzy matching (see `pipeline.eval()`'s `context_matching` params).
+    # - `document_id`: The answer is only considered as correct if its document ID matches as well. You can specify a custom document ID through `pipeline.eval()`'s `custom_document_id_field` param.
+    # - `document_id_and_context`: The answer is only considered as correct if its document ID and its context match as well.
+    #
+    # In Question Answering to get the very same definition of correctness for document metrics as for answer metrics, `document_scope` must be 'answer' or 'document_id_or_answer'.
```

Comment:
At first, it was difficult to me to understand this sentence. What does `very same definition of correctness` mean? Something like "The retrieved document is evaluated as correct whenever the answer is correct", right?

Reply:
Yes.

Reply:
done.